sdk/extensions/chronopost/karrio/providers/chronopost/tracking.py

```diff
@@ -0,0 +1,84 @@
+import typing
+from urllib import response
+import karrio.lib as lib
+import karrio.core.models as models
+import karrio.providers.chronopost.utils as provider_utils
+import karrio.providers.chronopost.error as provider_error
+import chronopost_lib.trackingservice as chronopost
+
+
+def parse_tracking_response(
+    responses: typing.List[lib.Element],
+    settings: provider_utils.Settings,
+) -> typing.Tuple[typing.List[models.TrackingDetails], typing.List[models.Message]]:
+    tracking_nodes: typing.List[chronopost.eventInfoComp] = [
+        lib.find_element("events", response, chronopost.eventInfoComp)
+        for response in responses
+    ]
```

Comment:
I would suggest getting the `listEventInfoComp` node instead because it is specific to one `skybillnumber` and gives you the events too.
```suggestion
    tracking_nodes: typing.List[chronopost.eventInfoComp] = [
        lib.find_element("listEventInfoComp", response, chronopost.eventInfoComp)
        for response in responses
    ]
```

Reply:
I was going with this approach at first, but when I tried to get an attribute like eventDate from listEventInfoComps.events, nothing shows up in the editor. It lured me into thinking I was using the wrong classes 😅 but apparently it works.